x/oracle/types/errors.go

```diff
@@ -10,17 +10,18 @@ import (
 // Oracle sentinel errors
 // nolint: lll
 var (
-	ErrInvalidExchangeRate   = sdkerrors.Register(ModuleName, 2, "invalid exchange rate")
-	ErrNoPrevote             = sdkerrors.Register(ModuleName, 3, "no prevote")
-	ErrNoVote                = sdkerrors.Register(ModuleName, 4, "no vote")
-	ErrNoVotingPermission    = sdkerrors.Register(ModuleName, 5, "unauthorized voter")
-	ErrInvalidHash           = sdkerrors.Register(ModuleName, 6, "invalid hash")
-	ErrInvalidHashLength     = sdkerrors.Register(ModuleName, 7, fmt.Sprintf("invalid hash length; should equal %d", tmhash.TruncatedSize))
-	ErrVerificationFailed    = sdkerrors.Register(ModuleName, 8, "hash verification failed")
-	ErrRevealPeriodMissMatch = sdkerrors.Register(ModuleName, 9, "reveal period of submitted vote does not match with registered prevote")
-	ErrInvalidSaltLength     = sdkerrors.Register(ModuleName, 10, "invalid salt length; should be 1~4")
+	ErrInvalidExchangeRate   = sdkerrors.Register(ModuleName, 1, "invalid exchange rate")
+	ErrNoPrevote             = sdkerrors.Register(ModuleName, 2, "no prevote")
+	ErrNoVote                = sdkerrors.Register(ModuleName, 3, "no vote")
+	ErrNoVotingPermission    = sdkerrors.Register(ModuleName, 4, "unauthorized voter")
+	ErrInvalidHash           = sdkerrors.Register(ModuleName, 5, "invalid hash")
+	ErrInvalidHashLength     = sdkerrors.Register(ModuleName, 6, fmt.Sprintf("invalid hash length; should equal %d", tmhash.TruncatedSize))
+	ErrVerificationFailed    = sdkerrors.Register(ModuleName, 7, "hash verification failed")
+	ErrRevealPeriodMissMatch = sdkerrors.Register(ModuleName, 8, "reveal period of submitted vote does not match with registered prevote")
+	ErrInvalidSaltLength     = sdkerrors.Register(ModuleName, 9, "invalid salt length; must be [60, 80]")
```

Comment:
Required the salts to be [30 bytes, 40 bytes] - we could restrict it to something specific like 32 bytes if we want, wdyt @alexanderbez ?

Reply:
Yes, Can we just restrict to a single byte size value? like 32